sechub-wrapper-prepare/build.gradle

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+/*============================================================================
+* Build file for subproject
+*
+* Root build file: "${rootProject.projectDir}/build.gradle"
+* ============================================================================
+*/
+dependencies {
+    /* runtime */
+    implementation project(':sechub-commons-archive')
+    implementation project(':sechub-commons-model')
+    implementation project(':sechub-commons-core')
+    implementation project(':sechub-commons-pds')
+
+    implementation project(':sechub-adapter-checkmarx')
+
+    /* test */
+    testImplementation project(':sechub-testframework')
+
+    testImplementation spring_boot_dependency.junit_jupiter
+    testImplementation spring_boot_dependency.junit_jupiter_params
+    testImplementation spring_boot_dependency.mockito_core
+
+    testImplementation library.apache_commons_io
+}
+
+version = versionData.getPrepareWrapperVersion()
+
+bootJar {
+
+    doLast {
+        /* Here we copy the created wrapper jar as "sechub-wrapper-prepare.jar" into the
+         * pds-tools folder inside the sechub-integrationtest gradle subproject.
+         */
+        File integratonTestLaunchFile = new File("${rootProject.projectDir}/sechub-integrationtest/build/pds-tools/sechub-wrapper-prepare.jar")
```

Comment:
Okay, but the purpose should be clear: This is copied from checkmarx wrapper parts and the reason why this was done is, that the checkmarx wrapper is integration tested!

When we do this here as well, we could write an integration test for the prepare wrapper as well!

How?

First we write a special class:

```java
@Component
@Profile(PrepareWrapperModuleProfiles.INTEGRATIONTEST)
public class IntegrationTestPrepareWrapperModule implements PrepareWrapperModule

   @Autowired
   StorageService storageService;

   public boolean isAbleToPrepare(...){
     return true;
   }

   public void prepare(SecHubConfiguration model, ... more) throws IOException{
      String someOutputData = """ 
         ... some test data which will be checked inside integration test
         maybe with additional information - e.g. a variable output for the 
         used test credentials etc
      """;
      storageService.write(someOutputData);
   } 
```

Add the PDS wrapper script from the PDS-solution into the integration test PDS configuration file.

After this we write a "normal" SecHub integration test (with a new scenario + profile were a prepare wrapper is configured;

Last but not least: write an integration test which uses the new profile + checks the expected output data